app/helpers/application_helper/button/vm_native_console.rb

```diff
@@ -0,0 +1,16 @@
+class ApplicationHelper::Button::VmNativeConsole < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == 'redhat'
+  end
+
+  def disabled?
+    begin
+      @record.validate_native_console_support
```

Comment:
Can you confirm that this`validate_native_console_support` only talks to the database? (it does not do any time consuming operation and does to try to access the VM or talk to a provider).

Thx!

Reply:
Hi @martinpovolny, yes, it's testing only if the ext_management_system is nil and the state is on.
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/c544028e4aea14910c0a9ebacb02107dd4156f98/app/models/manageiq/providers/redhat/infra_manager/vm/remote_console.rb#L43-L49